components/automate-chef-io/content/docs/iam-v2-api-reference.md

```diff
@@ -10,291 +10,788 @@ toc = true
+curl -sH "api-token: $TOKEN" \
+  https://{{< example_fqdn "automate" >}}/apis/iam/v2beta/policies?pretty
 ```
 
-The output includes both `Chef-managed` and `custom` policies.
-Policies migrated from IAM v1 are included as `custom` policies.
-Policy names that begin with `[Legacy]` are default policies migrated from v1.
-We recommend deleting these legacy policies and setting up new IAM v2 policies in their place.
+For those API methods that take JSON data (typically `create` and `update` methods)
+you can provide that data to the REST endpoint in a variety of ways.
+If it is a relatively small size payload, you can include it in a `curl` command inline, e.g.
 
-In the browser: Navigate to the *Policies* page under the **Settings**  tab.
-
-### Getting a Policy
+```bash
+curl -sH "api-token: $TOKEN" -d '<your JSON here>' ...
+```
 
-In order to fetch a policy on the command-line, you must know its ID.
-This may be obtained by first fetching the list of all policies; see [Listing Policies]({{< relref "iam-v2-api-reference.md#listing-policies" >}}).
-Find the `id` field for the policy of interest--for example, the ID from the default administrator policy is `administrator-access`.
-Plug that in to the URL as shown:
+If the payload is larger, it is often convenient to store it in a file, then pass that file, e.g.,
 
 ```bash
-curl -s -H "api-token: $TOKEN" https://{{< example_fqdn "automate" >}}/apis/iam/v2beta/policies/administrator-access?pretty
+curl -sH "api-token: $TOKEN" -d @policy.json ...
 ```
 
-In the browser: Navigate to the *Policies* page under the **Settings**  tab, and then select an individual policy to open its details.
+## Projects Property
 
-### Creating a Policy
+Teams, tokens, roles, and policies all contain a top-level `projects` property; you will see that in the example JSON for each resource in the following sections.
+This property associates the particular IAM resource with one or more projects.
+Users must have permissions for those projects to be able to view or modify those resources.
+If the `projects` property is left empty, that resource is categorized as *unassigned*.
+
+To create those permissions for users, there is a separate and distinct `projects` property on each policy statement.
+This `projects` property may **not** be empty; you must either specify specific projects or use a wildcard (`*`) to indicate all projects.
+
+Type               | May be empty ?    | Allows "*" ?         | May specify (unassigned) ?
+-------------------|-------------------|----------------------|---------------------------
+top-level projects | empty allowed     | wildcard not allowed | (unassigned) not allowed
+statement projects | empty not allowed | wildcard allowed     | (unassigned) allowed
+
+## Policies
+
+HTTP request              | Description
+--------------------------|------------
+GET /policies             | lists both *Chef-managed* and *Custom* policies
+GET /policies/***id***    | gets the specified policy
```

Comment:
Could we change `***id***` to `{id}` ? Just to keep the conventions for values substitutions consistent?

Example:
`GET /policies/{**id**}` 
(which would keep the bold. That's a nice touch)

Reply:
yes of course, will do.

Reply:
Thanks!